### ISSUE TITLE
3.2 Polish Translation crashes settings window

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplate4#104]
There is an error in the polish translation leading to the settings window not being able to open.

![image](https://user-images.githubusercontent.com/6314674/102794516-65b4a580-43a3-11eb-8f9c-e705a779ca51.png)

This was caused by a missing double quote in the retranslated line 172:
```
<!-- address field switches -->
<!ENTITY spec_form.cap "Przełączniki pola adresu">
```

below is a patched version:

[smartTemplate-fx-3.2.1pre16.zip](https://github.com/RealRaven2000/SmartTemplate4/files/5725200/smartTemplate-fx-3.2.1pre16.zip)

to install, download the zip file and drag it into Thunderbird Add-ons Manager.

**Comment by RealRaven2000** [comment]
More translation fixes (and some new items, especially #105) were added in this latest prerelease:

[smartTemplate-fx-3.3pre22.zip](https://github.com/RealRaven2000/SmartTemplate4/files/5751058/smartTemplate-fx-3.3pre22.zip)

to install, download the zip file and drag it into Thunderbird Add-ons Manager.

**Comment by RealRaven2000** [comment]
Fixed in 3.3 released for Thunderbird 78 on 03/01/2021
